src/app/profile/email-verification/email-verification.component.html

```diff
@@ -0,0 +1,44 @@
+<div  class="container f8-email-verify">
+  <div class="row">
+    <div class="blank-slate-pf">
+      <div class="rh-branding-logo">
+        <img class="width-100"
+          src="../../../assets/images/Logotype_RH_OpenShift-io_RGB_RedGray.png">
+      </div>
+      <div *ngIf="verified === 'true'">
+        <img 
+          width="100"
+          height="100" 
+          src="../../../assets/images/trophy.png" />
+        <h1>
+          Your e-mail has been confirmed
+        </h1>
+        <hr>
+        <p>
+          Thank you for validating your e-mail address.
+          You can now continue to use Openshift.io
+        </p>
+      </div>
+      <div *ngIf="verified === 'false'">
+        <img
+          width="100"
+          height="100"
+          src="../../../assets/images/neutralface.png">
+        <h1>
+          This URL has already been used to verify your e-mail
+        </h1>
+        <hr>
+        <p>
+          Your e-mail has already been approved.
+          Click on my profile below to continue.
+        </p>
+      </div>
+      <div class="blank-slate-pf-main-action">
+        <a href="#"
```

Comment:
This should be `<button>`, being that it is a button. If it needs a clickable action, that can be handled by a `(click)="event()"` event.